addons/project/models/project.py

```diff
@@ -451,8 +451,8 @@ def _read_group_stage_ids(self, stages, domain, order):
         ('blocked', 'Red')], string='Kanban State',
         copy=False, default='normal', required=True)
     kanban_state_label = fields.Char(compute='_compute_kanban_state_label', string='Kanban State Label', track_visibility='onchange')
-    create_date = fields.Datetime(index=True)
-    write_date = fields.Datetime(index=True)  #not displayed in the view but it might be useful with base_automation module (and it needs to be defined first for that)
+    create_date = fields.Datetime("Created On", readonly=True, index=True)
```

Comment:
why in those cases not just putting `index=True`? The string and the `readonly` attributes will be inherited from the base model, right? 🤔

Reply:
@mreficent the log_access fields are **not** on the base model. The reason is that you can choose to not have them by setting `_log_access = False` on your model's class.

In practice, the automatic field `create_uid` (resp. `create_date`, `write_uid`, `write_date`) is added to the model's final class only if it is not defined yet.

Reply:
Thanks for your answer :)